examples/cubic_planar.html

```diff
@@ -61,8 +61,7 @@
         <script type="text/javascript">
             window.THREE = itowns.THREE;
         </script>
-        <script src="../node_modules/three/examples/js/controls/OrbitControls.js"></script>
-        <script src="GUI/dat.gui/dat.gui.min.js"></script>
+        <script src="https://cdn.rawgit.com/mrdoob/three.js/dev/examples/js/controls/OrbitControls.js"></script>
```

Comment:
Maybe use https://unpkg.com/three@0.86.0/examples/js/controls/OrbitControls.js instead? (or least choose a version that matches our version of three.js, just in case)